src/redux/books/books.js

```diff
@@ -0,0 +1,33 @@
+import { useReducer, useState } from 'react';
+
+const ACTIONS = {
+  ADD_BOOK: 'add-book',
+  REMOVE_BOOK: 'remove-book',
+};
```

Comment:
- [ ] **Required:** Reducer action types are required to be defined following [the ducks' pattern.](https://github.com/erikras/ducks-modular-redux) Kindly follow this pattern.